metaflow/plugins/argo/test/test_run_jsontype.py

```diff
@@ -0,0 +1,46 @@
+from metaflow.client.trigger_live_run import trigger_live_run  # trigger
+import time
+
+
+flow_name = None  # "HelloArgoFlowTwo", "HelloJSONType", None
+template_name = "hellojsontype"  # "helloargoflowtwo", "hellojsontype", None
+
+parameters_valid = {
+    "str_to_list": "[1, 2, 3, 4, 5]",
+    "str_to_dict": '{"c": [1, 2, 3], "d": 99, "is_input_param": "yes"}',
+    "list_param": ["this", "is", "a", "list"],
+    "dict_param": {"e": [4, 5, 6], "f": 22, "starts_as_dict": "yes"},
+    "reg_str": "this is a string",
+    "date_key": "2020-01-01",
+    "integer": 1,
+    "boolean": True,
+    "float": 123.456,
+}
```

Comment:
Can you also add these parameters into the `hello_jsontype.py` flow so that they are tested?

Reply:
I added a TODO in the test file and test flow to remind me to do this as part of [AIP-6471](https://zbrt.atl.zillow.net/browse/AIP-6471) (Automate integration tests for FTF feature)